modules/ossm-rn-known-issues.adoc

```diff
@@ -25,6 +25,7 @@ These limitations exist in {SMProductName}:
 
 * WebAssembly extensions are not supported on IBM Z and IBM Power Systems.
```

Comment:
Remove "Systems": 

* WebAssembly extensions are not supported on IBM Z and IBM Power.

Reply:
Just noticed the Bookinfo restriction (line 24) also says IBM Power Systems. Remove Systems there as well.

Reply:
Thanks, @SNiemann15! I've made this change.